src/blocks/modal.css

```diff
@@ -29,6 +29,14 @@
   background-color: #fff;
 }
 
+.modal__container_confirm {
+  max-height: 181px;
+}
+
+.modal__container_avatar {
+  max-height: 272px;
+}
+
 .modal__close {
```

Comment:
Needs correction:

1) The cursor of a disabled button should not be `pointer`, use `default`

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

2) Disabled button should not have `hover` state. 

There are several ways to fix that

1) use `not` selector in `.modal__button:hover` styles

```css
.modal__button:hover:not(:disabled) { // selects not disabled hovered button
    opacity: 0.8;
    cursor: pointer;
}
```

2) override hovered button styles (this rule should be after `.modal__button:hover`)

```css
.modal__button_disabled:hover { // selects disabled hovered button
    opacity: 1;
    cursor: default;
}
```